README.md

```diff
@@ -52,6 +52,47 @@ npm install @google-cloud/recaptcha-enterprise
 ```
 
 
+### Using the client library
+
+```javascript
+/**
+ * Instantiate the reCAPTCHA Enterprise client and make request.
+ * @param {string} projectNumber The project number to use when making requests.
+ */
+async function quickstart(projectNumber) {
+  // const projectNumber = [PROJECT_NUMBER];
+
+  // Create the reCAPTCHA client library.
+  const {
+    RecaptchaEnterpriseServiceV1Beta1Client,
+  } = require('@google-cloud/recaptcha-enterprise');
+  const client = new RecaptchaEnterpriseServiceV1Beta1Client();
+
+  // format the path to the project, projectNumber should be
+  // provided.
+  const formattedParent = client.projectPath(projectNumber);
```

Comment:
Just get the `projectId` using `const projectId = await client.getProjectId();` and lose the parameter